go.mod

```diff
@@ -60,6 +62,7 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/gnostic v0.5.7-v3refs // indirect
 	github.com/google/go-github/v45 v45.2.0 // indirect
+	github.com/google/go-github/v52 v52.0.0
```

Comment:
Thank you @th-le! TIL this had to be bumped in order to accommodate https://github.com/google/go-github/commit/9385ab02991f311db7d6be6a67717d0ff50fc84f.
However, I'm still unsure why this had to be in this second block of `require`.
Shouldn't it have resulted in updating L13 of this file?!